### DISCUSSION TITLE
Sunshine Black screen when screen saver activates/ RivaTuner Bug

### DISCUSSION THREAD

**Discussion by agustinsau** [LizardByte/community#212] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

First, Ive notice that when using sunshine, in the first second you open Rivatuner in the front window (everything ok if its running in the background), sunshine lose control with the host pc. The connection still functionable but not its control with the mouse. 
Secondly, if the screen is not turned on (in the HOST pc) or the screen saver activates so it turn off the screen... sunshine shows a black screen too and there is no way to sleep it out.

### Expected Behavior

Maybe mantain control over the remote pc when the screen goes off like in Team viewer?

### Host Operating System

Windows

### Operating System Version

windows 11

### Architecture

64 bit

### Sunshine commit or version

v0.19.1 

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

gtx 3080

### GPU Driver/Mesa Version

last version sorry cant remember v, number

### Config

```Shell
default .conf config
```

### Apps

```json
Desktop remote control, no apps.
```

### Relevant log output

```shell
...
```

**Comment by RealZack** [comment]
I'm interested in this too, I have two computers connected to the same screen, I'm using moonlight to control the second one.

Most of the time the screen is just black, I have to change the input of the lcd to go on my second computer, it's "waking up" the video, then moonlight is working perfectly fine from my first computer.

May a prepcmd could do the trick, didn't find any process to "wake" video

**Comment by Peristroff** [comment]
+1 to this. Turning the screen off could lead to a better battery life in notebooks. Currently, shutting down the screen only delivers a black screen to Moonlight, but the controls and sounds works perfectly.

>Maybe mantain control over the remote pc when the screen goes off like in Team viewer?

TeamViewer does some black magic to make the screen work while is off and I would like for Sunshine to implement this.

Note aside, this (at least theoretically) has the potential to make some crazy setups, like carring a gaming notebook in a backpack while playing in your phone (I wouldn't recommend this because of airflow, but the possibility is there).

**Reply by ReenigneArcher** [reply]
https://www.reddit.com/r/ProgrammerHumor/comments/sgwew9/how_qas_test_developers_code/

[a comment by an automated account is not shown]
